src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -374,11 +375,16 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
             return validationInfo;
         }
 
-        internal void ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto)
+        internal ModelValidationNode ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto)
```

Comment:
>Maybe just return the MBR here?

Except for the fact that there is no model binding happening here :smile: .
An alternative is I can pass in a Model ValidationNode as a parameter, if that makes it clearer.